l10n_ro_stock_report/report/stock_report_view.xml

```diff
@@ -0,0 +1,364 @@
+                <field name="cantitate_intrata" sum="1" />
+                <field name="valoare_intrata" sum="1" />
+                <field name="cantitate_iesita" sum="1" />
+                <field name="valoare_iesita" sum="1" />
+                <field name="cantitate_finala" sum="1" />
+                <field name="valoare_finala" sum="1" />
+                <field name="referinta" />
+                <field name="partener" />
+            </tree>
+        </field>
+    </record>
+
+     <template id="internal_layout">
+         <div class="article">
+            <link
+                href="/l10n_ro_stock_report/static/src/css/report.css"
+                rel="stylesheet"
+            />
+            <t t-raw="0" />
+        </div>
+        <div class="footer">
+            <div class="row">
+                <div class="col-6 custom_footer">
+                    <span
+                        t-esc="context_timestamp(datetime.datetime.now()).strftime('%Y-%m-%d')"
+                    />
+                </div>
+                <div class="col-6 text-right custom_footer">
+                    <span class="page" />/<span class="topage" />
+                </div>
+            </div>
+        </div>
+    </template>
+
+     <template id="report_stock_card">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="l10n_ro_stock_report.internal_layout">
+                    <t t-call="l10n_ro_stock_report.report_stock_card_report_base" />
+                </t>
+            </t>
+        </t>
+    </template>
+
+     <template id="report_stock_card_report_base">
+        <t t-foreach="o.found_product_ids" t-as="product">
+            <div class="page">
+                <div class="row">
+                    <t t-set="title">Stock Card - <t t-raw="product.name" /></t>
+                    <h4 class="mt0" t-esc="title" style="text-align: center;" />
+                </div>
+                <t t-call="l10n_ro_stock_report.report_stock_card_report_filters" />
+                <div class="act_as_table data_table mt4" style="width: 100%;">
+                    <t
+                        t-set="initial"
+                        t-value="o.line_product_ids.filtered(lambda l: l.product_id == product and l.referinta =='INITIALA')"
+                    />
+                    <t
+                        t-set="finala"
+                        t-value="o.line_product_ids.filtered(lambda l: l.product_id == product and l.referinta =='FINALA')"
+                    />
+                    <t t-call="l10n_ro_stock_report.report_stock_card_lines_header" />
+                    <!-- Display initial lines -->
+                     <div class="act_as_row lines">
+                        <div class="act_as_cell" />
+                        <div class="act_as_cell">
+                            INITIAL
+                        </div>
+
+                        <div class="act_as_cell right">
```

Comment:
ar treui merge-uite sa afisam in ultimele 2 coloane, acum e  un pic ciudat, inital e in coloanele cu intrari si final e sub iesiri...